src/Ng/Feed2Catalog.cs

```diff
@@ -240,6 +354,49 @@ private static DateTime ForceUTC(DateTime date)
             return lastDate;
         }
 
+        static async Task<DateTime> Deletes2Catalog(SortedList<DateTime, IList<Tuple<string, string>>> packages, Storage storage, DateTime lastCreated, DateTime lastEdited, DateTime lastDeleted, bool? createdPackages, CancellationToken cancellationToken)
```

Comment:
switch SortedList<DateTime, IList<Tuple<string, string>>> to an object model, it is unreadable and unmaintanable